scalajvm/app/views/meta/quoteEntity.scala.html

```diff
@@ -0,0 +1,7 @@
+@(quote: models.data.Quote)(implicit requestHeader: play.api.mvc.RequestHeader)
+
+@entity(s"Log List ⚙ Цитата #${quote.id}",
+        "loglist:quote",
+        routes.Assets.at("images/gear_400x400.png").absoluteURL(),
```

Comment:
I meant I'll research and **not** create a PR for that because of [reasons](https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images).